### PR TITLE
docs: add verfiy RTMR with replayed event log related content

### PR DESCRIPTION
**Pull request by Hulongyin**
This PR is to add verify RTMR with replayed event log related content to the `How_to_Protect_AI_Models_in_Cloud_Native_Environments` documentation.

### CODE REVIEW

File: docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md [hunk 1]
```diff
@@ -10,8 +10,9 @@ To ensure the cloud native environments and AI models are secure protected in us
 
 ## 1. Architecture Design
 
-This design can be divided into three steps logically.
+This design can be divided into four steps logically.
 - A plain AI model is preprocessed before uploading or use in cloud native environment. AI model should be encrypted offline, and the encryption key will be stored in a key database server.
+- Fetch and verify event logs. Event log is the record of measurements made to PCRs by the Platform Firmware, with some informational events not extended to PCRs. The informational events are used to convey valuable information to an evaluator of the log. Each measurement made, as well as the information events, are recorded in the event log as individual entries with specified fields. Event logs can assist the attestation/verification process. For more detail, please refer to [TCG_PCClient Spec](https://trustedcomputinggroup.org/wp-content/uploads/TCG_PCClientSpecPlat_TPM_2p0_1p04_pub.pdf).
```
Comment (leyao-daily) [inline docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md]: PCRs?

Comment (leyao-daily) [reply docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md]: Event logs or event log.

File: docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md [hunk 2]
```diff
@@ -36,20 +37,38 @@ To protect the model and its encryption key, the following preprocessing steps a
 
 The service supports attestation, measurement fetching and event log collecting of various platforms including Intel TDX, Trusted Platform Modules (TPM) and AMD Secure Encrypted Virtualization-Secure Nested Paging (SEV-SNP) (will be supported soon).
 
+CCNP is a good choice to get event log and it hides the complexity and is easy to use, sample code from CCNP:
```
Comment (Ruoyu-y) [inline docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md]: CCNP is a good choice to fetch these evidences including measurements and event logs, which hides the complexity of the underlying platforms and increase the usability of the APIs. Here's the sample code using CCNP:

File: docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md [hunk 3]
```diff
@@ -36,20 +37,38 @@ To protect the model and its encryption key, the following preprocessing steps a
...
+
+```Python
+from ccnp import Eventlog, EventlogType
```
Comment (Ruoyu-y) [inline docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md]: unused import "EventlogType"

File: docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md [hunk 4]
```diff
@@ -36,20 +37,38 @@ To protect the model and its encryption key, the following preprocessing steps a
...
+event_logs = Eventlog.get_platform_eventlog()
+```
+
+To verify that event log has not been tampered with, we can compare IMR (Integrated Measurement Register) value from CCNP fetching and replayed via event log.
```
Comment (Ruoyu-y) [inline docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md]: To verify that the event logs have not been tampered with, we can compare the measurement replayed from event logs with the IMR (Integrated Measurement Register) values fetched using CCNP.

File: docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md [hunk 5]
```diff
@@ -36,20 +37,38 @@ To protect the model and its encryption key, the following preprocessing steps a
...
 
-CCNP API detail documentation can be found [here](https://intel.github.io/confidential-cloud-native-primitives/).
-
-### 1.3 AI Model Decryption
+### 1.4 AI Model Decryption
 
 CNAP project’s Inference Service flow of getting AI model should be updated to support TEE environment since the AI model has been encrypted:
 
 - Deploy CCNP as a [DaemonSet](https://intel.github.io/confidential-cloud-native-primitives/_rst/service.quote.html#deploy-as-daemonset-in-kubernetes), it will detect the TEE environment.
+- Fetch and verify evnet log.
```
Comment (Ruoyu-y) [inline docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md]: Minor typo.  "evnet log" -> "event log"

File: docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md [hunk 6]
```diff
@@ -192,7 +212,44 @@ req_body = {
 }
 ```
 
-### 2.5 Attestation
+### 2.5 Fetch and Verify Event Log
+
+CCNP provides an easy way to fetch and verify event log in TEE. After fetching event log, we can compare IMR (Integrated Measurement Register) value from CCNP fetching and replayed via event log to verify that event log has not been tampered with.
```
Comment (Ruoyu-y) [inline docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md]: we can compare the measurement replayed from event logs with the IMR (Integrated Measurement Register) values fetched using CCNP to verify that event logs have not been tampered with.

File: docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md [hunk 7]
```diff
@@ -251,8 +308,8 @@ def decrypt_data(encrypted_data, key) -> bytes:
...
-This document describes a common architecture to “design and run” a native AI Model into confidential computing environments. (1) AI Model preprocess (2) Attestation of the cloud native environment (3) AI Model fetching and decryption.
-Intel’s TDX technology can provide a TEE running environment, and CCNP can simply the attestation of confidential native environment, last but not least, a sample KBS is introduced to help customer to connect with its own or vendor-independent attestation service (for example, Intel Trust Authority).
+This document describes a common architecture to “design and run” a native AI Model into confidential computing environments. (1) AI Model preprocess (2) Fetch and verify event logs (3) Attestation of the cloud native environment (4) AI Model fetching and decryption.
+Intel’s TDX technology can provide a TEE running environment, and CCNP can simply the event log, measurement fetching and attestation of confidential native environment, last but not least, a sample KBS is introduced to help customer to connect with its own or vendor-independent attestation service (for example, Intel Trust Authority).
```
Comment (Ruoyu-y) [inline docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md]: can simply -> an simplify

**Review by Ruoyu-y — CHANGES_REQUESTED** [review]

File: docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md (hunk 1 above)
Comment (Hulongyin) [reply docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md]: Thanks, fixed.

File: docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md (hunk 2 above)
Comment (Hulongyin) [reply docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md]: Thanks, all fixed.

**Review by dongx1x — APPROVED** [review]
LGTM.

**Pull request merged**